src/main/java/com/sun/tdk/signaturetest/core/ClassCorrector.java

```diff
@@ -701,7 +711,8 @@ private boolean isInvisibleClass(String fqname) {
 
         if (fqname.startsWith("?"))
             return false;
-
+        if (jdkExclude.isJdkClass(fqname)) 
+            return false;
```

Comment:
Is this basically to avoid ever loading fqname?

Reply:
That is a good question, I captured the Exception [here](https://gist.github.com/scottmarlow/24b613b056a6cd007e3a437b668b3a54), which shows one of the typical failures I was seeing:
```
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make public abstract java.lang.String jdk.internal.vm.annotation.Contended.value() accessible: module java.base does not "exports jdk.internal.vm.annotation" to unnamed module @2d2c4fbd
 	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340)
 	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280)
 	at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:198)
 	at java.base/java.lang.reflect.AccessibleObject.setAccessible(AccessibleObject.java:123)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.loaders.TigerRefgClassDescrLoader.parse(TigerRefgClassDescrLoader.java:475)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.loaders.TigerRefgClassDescrLoader.parse(TigerRefgClassDescrLoader.java:461)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.loaders.TigerRefgClassDescrLoader.readFields(TigerRefgClassDescrLoader.java:178)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.loaders.TigerRefgClassDescrLoader.readClass(TigerRefgClassDescrLoader.java:131)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.loaders.TigerRefgClassDescrLoader.load(TigerRefgClassDescrLoader.java:75)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.core.ClassHierarchyImpl.load(ClassHierarchyImpl.java:180)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.core.ClassHierarchyImpl.load(ClassHierarchyImpl.java:164)
 	at deployment.JavaEESigTest_ejb_vehicle.ear//com.sun.tdk.signaturetest.core.ClassHierarchyImpl.isAccessible(ClassHierarchyImpl.java:232)
```

Could that be solvable by JBoss Modules instead?

Reply:
I see. No, I don't think we could / should do anything with JBoss Modules or WildFly settings about this. SE rightly doesn't code poking at its internals.